src/Bicep.Core.Samples/Files/InvalidModules_LF/main.bicep

```diff
@@ -27,25 +27,62 @@ module moduleWithInterpPath './${interp}.bicep' = {
 
 }
 
+module moduleWithConditionAndInterpPath './${interp}.bicep' = if (true) {
```

Comment:
Can you also add cases that simulate typing the conditional progressively to make sure we don't lose syntax highlighting as the user is typing. I'm thinking cases like these (you have some of those):
- `module foo './foo.bicep' = if`
- `module foo './foo.bicep' = if (`
- `module foo './foo.bicep' = if (true`
- `module foo './foo.bicep' = if (true)`
- `module foo './foo.bicep' = if (true) {`
- etc.

Reply:
Same for resources also.

Reply:
Added the tests.